accounts/models.py

```diff
@@ -157,18 +158,22 @@ def studies(self):
 
     @cached_property
     def is_org_admin(self):
-        if not self.organization:
+        if not self.organization_id:
```

Comment:
`organization_id` is the field in the studies table in SQL that represents the ForeignKey to the organization. If you have an instance of the study object you know the value of that field without having to talk to the database or really do anything other than the value comparison in python.

The second you reference `self.organization` django's lazy ORM is going to do a query for an organization with an `id` of `organization_id`, then it's going to create an instance of the `Organization` model with all of that data it gets back. Just unnecessary for what you're trying to accomplish. Django's not smart enough to know that you just want to null-check that field, not use the org name or any other field.